Classes/KIFUIViewTestActor.m

```diff
@@ -6,11 +6,12 @@
 //
 //
 
-#import "KIFUIViewTestActor.h"
-#import "UIWindow-KIFAdditions.h"
-#import "NSPredicate+KIFAdditions.h"
-#import "UIAccessibilityElement-KIFAdditions.h"
-#import "NSString+KIFAdditions.h"
+#import <KIF/KIFUIViewTestActor.h>
```

Comment:
I think that using quotes should be the norm for inclusions within source files. If this is generating an error, I think it is something we should take a look at.